.github/workflows/AzureDeploy.yml

```diff
@@ -0,0 +1,28 @@
+name: Blob storage website CI
+
+on:
+  pull_request:
+    branches:
+      - development
+      - TestBranch
```

Comment:
Using on.pull_request will trigger this action whenever a PR is created, and not when it is completed/merged. Should this be on.push instead?

Reply:
Further question, should the test branch be included here?

Reply:
That was not the case when I tested this. It would Re-Deploy as soon as the PR was complete.